addons/escoria-core/game/core-scripts/esc/esc_action_manager.gd

```diff
@@ -540,6 +540,19 @@ func perform_inputevent_on_object(
 
 		if not player_global_pos.is_equal_approx(destination_position):
 			dont_interact = true
+			escoria.logger.warn(
```

Comment:
I'd argue this should be `info` and not `warn` as nothing is "potentially going wrong" in the system itself, just the game.

Reply:
I don't feel particularly strongly here so don't mind changing it. The reason I'd suggest it's a warning is that I've answered questions from new users who can't figure out why their interaction isn't working. They're more likely to see warning's than info messages. 
If you're happy that they'll go straight for debug when trying to work out a problem then I'm happy to make it info :-)

Reply:
Made it info as discussed in discord.